spec/models/like_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+
+feature 'Liking photos' do
+
+  context 'Signed in' do
+
+    let!(:sea) {Photo.create(title:'Awesome sea',
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.